git_day_1.adoc

```diff
@@ -0,0 +1,138 @@
+Git Gud Speaker Outline - Day 1
+-------------------------------
+:hardbreaks:
+
+The Basics
+~~~~~~~~~~
+The goal of this presentation is to give you an understanding of how *local* git works, and will not extensively
+cover the syntax and associated _how to_ around using git. Specific syntax questions can be resolved at the end
+of the presentation or via google.
+
+Understand Your Audience
+~~~~~~~~~~~~~~~~~~~~~~~~
+* Is there anyone that doesn't know git?
+* If you were on a client site, would you be comfortable teaching them about git?
+* What type of team environment did you use git in? Solo? Small Team? Army?
+
+Commits
+^^^^^^^
+* *Q*:What is a commit?
+** author, date, message, hash, parent
+* The importance of commit messages
+** Without a message, it's not a commit
+** Not too much, but enough to remind of whats been done
+* *Q*:What's the use of the hash?
+** Easily made unique identifier of a given set of changes
+* *Q*:What is the importance of the parent?
+** Tracks the list of previous changes
+** Creates a traversable linked list series of traversable changes
+* *Q*:Why are commits immutable?
+** Commits as cattle, not pets
+** History as something that shouldn't be changed
+* Commits are the *atomic unit* in git
+
+*DEMO STARTS HERE*
+(whiteboard or link:asciidoc[https://git-school.github.io/visualizing-git/])
+
+*DEMO*
+_set up a quick series of linear commits_
+
+Branches
+^^^^^^^^
+* *Q*: Can anyone tell me, technically or conceptually, what a branch is?
+** *Technically* - just a pointer
+** *Conceptually* - a series of commits starting from a given point in history
+** Super lightweight and easy to make and move
+** Describe use cases
+*** Sharing code
+*** Distributed work
+*** etc.
+
+*DEMO*
+_perform git branch feature_
```

Comment:
```suggestion
Run `git branch feature`
```

Reply:
I chose to go with _simulate_ here, since this could be performed as a whiteboard exercise